### PR TITLE
[ru] Localize digital-certificate.md

### PR DESCRIPTION
**Pull request by vickylikh**
### Describe your changes

This PR adds RU localization of digital certificate

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by tym83 — APPROVED** [review]
LGTM

File: content/ru/digital-certificate.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Digital Certificate
+status: Feedback Appreciated
+category: technology
+tags: ["security", "", ""]
+---
+
+(Цифровой) сертификат / сертификат открытого ключа / SSL-сертификат — это электронный документ, который используется для обеспечения безопасного обмена информацией по сети.
```
Comment (kirkonru) [inline content/ru/digital-certificate.md:8]:
```suggestion
(Цифровой) сертификат / сертификат открытого ключа / SSL-сертификат — это электронный документ, который используется для безопасного обмена информацией по сети. 
```

File: content/ru/digital-certificate.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Digital Certificate
+status: Feedback Appreciated
+category: technology
+tags: ["security", "", ""]
+---
+
+(Цифровой) сертификат / сертификат открытого ключа / SSL-сертификат — это электронный документ, который используется для обеспечения безопасного обмена информацией по сети. 
+Сертификат подтверждает, что объект, с которым происходит коммуникация, — действительно тот, за кого себя выдает.
```
Comment (kirkonru) [inline content/ru/digital-certificate.md:9]:
```suggestion
Сертификат подтверждает, что объект, с которым происходит коммуникация, действительно тот, за кого себя выдает.
```

File: content/ru/digital-certificate.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Кроме того, благодаря шифрованию исходящих и входящих данных сертификат гарантирует конфиденциальность обмена информацией.
+
+## Какую проблему решает
+
+Когда устройство взаимодействует по сети с другими сущностями, у него нет гарантии, что другое устройство, с которым оно взаимодействует, является тем, за кого себя выдает.
+Кроме того, нельзя гарантировать, что трафик между двумя устройствами не перехватит третья сторона.
+Следовательно, потенциально любая коммуникация может быть перехвачена, а значит, конфиденциальная информация, например имена пользователей и пароли, может быть скомпрометирована.
```
Comment (kirkonru) [inline content/ru/digital-certificate.md:16]:
```suggestion
Следовательно, любая коммуникация потенциально может быть перехвачена, а значит, конфиденциальная информация, например имена пользователей и пароли, может быть скомпрометирована. 
```

**Review by kirkonru — APPROVED** [review]

**Comment by kirkonru** [comment]
@vickylikh сongrats on your first PR to the glossary! 🥇

**Pull request merged**